l3kernel/testfiles/m3str002.lvt

```diff
@@ -56,5 +56,26 @@
     \tl_if_eq:NNTF \l_tmpa_tl \l_tmpb_tl \TRUE \FALSE
   }
 
+\OMIT
+\tl_set:N \l_tmpa_tl { a }
+\tl_set:N \l_tmpb_tl { a }
```

Comment:
```suggestion
\tl_set:Nn \l_tmpa_tl { a }
\tl_set:Nn \l_tmpb_tl { a }
```

Reply:
Hmm the two `\tl_set:N ...` didn't fail the checks because they are in between `\OMIT ... \TIMO`, hence the corresponding error lines were filtered out by `l3build`.  Less robust than I imagined.